src/util/http/websocket/WebSocketSession.cpp

```diff
@@ -70,10 +70,6 @@ net::awaitable<void> WebSocketSession::acceptAndWait(
 
     co_await ws_.async_accept(request, boost::asio::use_awaitable);
 
-    auto strand = net::make_strand(ws_.get_executor());
-
-    co_await net::dispatch(strand, net::use_awaitable);
-
```

Comment:
Is the reason here, that we expect the calling code to already run on a strand, and this is enforced because this method is private?